_data/simple-icons.json

```diff
@@ -840,6 +840,11 @@
             "hex": "A22430",
             "source": "https://www.gnu.org/graphics/social.html"
         },
+        {
+            "title": "GOG",
```

Comment:
Seems like they prefer "GOG.com" when referring to their website (which is what the logo is for). I haven't seen them use plain "GOG".

Reply:
Updated (also in the SVG `<title>`) 👍